packages/kbn-eslint-config/.eslintrc.js

```diff
@@ -297,5 +298,6 @@ module.exports = {
     '@kbn/imports/uniform_imports': 'error',
     '@kbn/imports/no_unused_imports': 'error',
     '@kbn/imports/no_boundary_crossing': 'error',
+    '@kbn/telemetry/event_generating_elements_should_be_instrumented': 'warn',
```

Comment:
```suggestion
    
...(!RUNNING_IN_FIX_MODE && {
      '@kbn/telemetry/event_generating_elements_should_be_instrumented': 'warn'
    }),
```

Reply:
Adding this plus something like `const RUNNING_IN_FIX_MODE = process.argv.includes('--fix');` at the top of the file should avoid CI to auto fix things around this rule